docs/objects-in-directional-light.html

```diff
@@ -3,9 +3,9 @@
 <body>
```

Comment:
This example in fact points out what looks to me to be a bug. The cuboid edges are not outlines like they are in the polygon.

Reply:
I don't know if this can be called a bug. This is expected as the edges of cuboids are drawn in the fragment shader, and the fragment shader is only executed for pixels inside the cuboid, but as the edges are just a few pixels thick, it is very hard to see the difference. I think this worths the performance gains.